src/ovl_1C_GrabBag/1837D0.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _1837D0_H_
+#define _1837D0_H_
+
+unkObjectStruct* D_800EDE70[6];   // Array of unkObjectStruct pointers?
+u16 D_800EE984;                   // Offst into D_800EDE70 array.
+void func_800F6C98_183E88();
```

Comment:
For functions that take no args, you should include `void` in the signature. If the args are left blank it means the function takes a variable amount of args